### PR TITLE
Fix for issue 152

### PR DESCRIPTION
**Pull request by evelynmitchell**
deleted quality.yml

### CODE REVIEW

**Comment by evelynmitchell** [comment]
I've also added the fix for 156 the codacy scan  to this PR

**Comment by evelynmitchell** [comment]
Added the fix for issue 159, which changes the README, example.py and sequential_workflow_example.py to use keys from the environment.

**Comment by evelynmitchell** [comment]
Corrected anthropic in example.py

**Comment by evelynmitchell** [comment]
You will need to manually fix the conflict.

File: swarms/models/layoutlm_document_qa.py
Comment (kyegomez) [inline swarms/models/layoutlm_document_qa.py:1]: this needs to be deleted

File: example.py
Comment (kyegomez) [inline example.py:1]: this needs to be deleted

Comment (evelynmitchell) [reply example.py:1]: I've done this.

File: swarms/models/layoutlm_document_qa.py
Comment (kyegomez) [reply swarms/models/layoutlm_document_qa.py:1]: there are errors from transformer

**Pull request closed without merging**